scripts/setup-dev.js

```diff
@@ -2,7 +2,14 @@
 const fs = require('fs-extra')
 const path = require('path')
 
-const target = path.resolve('../vets-website/build/localhost/generated')
+// No __dirname here because we want it to reach the adjacent folder outside of next-build
+const target = path.resolve(
+  '..',
+  'vets-website',
+  'build',
+  'localhost',
+  'generated'
+)
 const symlinkPath = path.resolve(__dirname, '..', 'public', 'generated')
 const cmsDataPath = path.resolve(__dirname, '..', 'public', 'data', 'cms')
```

Comment:
Thanks for this change and, especially, the comment attempting to explain what's going on. I'm happy to move forward on this, but I am adding a bit more commentary below in an attempt to clarify my initial line of questioning.

_WARNING: Theoretical discussion. It's certainly ok to just move on with life and not read this :)_

I immediately see how `target` is different from the other two. It's a directory outside `next-build` whereas the other two are directories within `next-build`. So it's not immediately crazy to me to see two different treatments divided by this line of separation. But I'm still not sure it makes sense to me. I'm not sure that difference should determine the use (or lack of use) of `__dirname`.

To me, the question of the usage of `__dirname` as it relates to path resolution boils down to whether it makes more sense to provide "directions" about the "address" of a particular resource relative to a certain directory, or relative to the root. If we use `__dirname`, we are telling the thing that needs those directions to the address that it makes most sense to start from the directory of the script being executed and go from there. This makes a ton of sense when the resource is a subdirectory of the directory of the current script because the relationship will never be independent. If the directory of the current script got moved down a few levels, things would still work. In general, this makes a lot of sense when the physical relationship (in the directory structure) _between_ the two resources is constant.

On the other hand, giving directions from the root makes sense when the resource in question isn't necessarily related to the current directory. In this case, I think it's arguable that this case is the more accurate. We know that `public` should always be a direct child of the root of the project. We cannot say the same about the location of the directory that houses `setup-dev.js`. To demonstrate, let's consider what would happen if we decided to add more scripts, and so we introduced another directory level:
```
- /public
  - /generated 
- /scripts
  - /dev
    - setup-dev.js
    - some-other-dev-script.js
  - /staging
    - setup-staging.js
  - /prod
    - setup-prod.js 
``` 
In this case, the code would have to be changed to accommodate. Sure, if we decided to move `public` to a different level, we'd have to change the code, but we'd have to change the code _in both cases_ (using `__dirname` or not).

**TL;DR** All three of these directory locations feel like they are best described by their directory location relative to the root of `next-build`, rather than to the directory of the current script.

Reply:
this makes a lot of sense, i've added __dirname to target as well. thank you.